fdbclient/AutoPublicAddress.cpp

```diff
@@ -28,7 +28,7 @@
 
 #include "fdbclient/CoordinationInterface.h"
 
-IPAddress determinePublicIPAutomatically(ClusterConnectionString const& ccs) {
+IPAddress determinePublicIPAutomatically(ClusterConnectionString& ccs) {
```

Comment:
Why remove const? Nothing seems changing ccs in this function.

Reply:
Yes, but we will need to call ccs.resolveHostnamesBlocking() in a followup PR.